Resources/Prototypes/Catalog/Fills/Crates/service.yml

```diff
@@ -74,3 +74,16 @@
     - id: GroundTobacco
       amount: 4
     - id: Matchbox
+
+- type: entity
+  id: CrateServiceBureaucracy
+  name: bureaucracy crate
+  description: Several stacks of paper and a few pens, what more can you ask for.
+  parent: CrateGenericSteel
+  components:
+  - type: StorageFill
+    contents:
+    - id: Paper
+      amount: 15
```

Comment:
This seems like a lot of entities that can be purchased for $500? Do these spawn as stacked?

Reply:
Paper isn't stacked at the moment, I didn't think they were able to be but I'll check that out shortly. I figured if someone's ordering paper they're using it for making fliers to hand to people or 'ticketing' people as security. If you have a better value I can change it.

Reply:
increased the cost to 1000